### DISCUSSION TITLE
Woensdag  11-12-24

### DISCUSSION THREAD

**Discussion by Nazneen05x** [rutgerkock/buurtcampus-oost#18] [Stand-ups]
- Hoe gaat het met je en waar ben je nú mee bezig?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by Nazneen05x** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?

Ik zit helaas in een dip en mijn energielevel is best laag. Ik ervaar best wel wat stress, want er moet toch nog wat veel dingen gebouwd en verbeterd worden. Ik heb momenten dat ik overzicht heb en lekker ga en dan heb ik momenten dat ik in stress raak omdat de dagen snel gaan en mijn energielevel niet wil meewerken.

### Wat wil je deze week bereiken (aan werken)?

Ik ben maandag en gisteren vooral bezig geweest met testen (over pagina), mails sturen en reageren op een pull request. Vandaag wil ik het coderen weer oppakken en beginnen met het verbeteren en code netjes maken van het menu.

### Waar loop je tegen aan?

Stress, dagen gaan snel en nog steeds bugs van het menu.

**Comment by Annevd** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?
Ik ben gister best een eind gekomen met de creative coding opdracht, waardoor ik tijd had om nog verder te werken aan het projectplan. Ik heb de puntjes op de i gezet van issue #216 waarvan de pull request hopelijk nu gemerged kan worden. 
### Wat wil je deze week bereiken (aan werken)?
Ik wil eigenlijk de contact pagina afmaken, zodat ik dat uit me hoofd kan zetten en kan focussen op de zaden pagina. Of dat gaat lukken moet nog wel blijken, aangezien ik ook nog wel bezig ben met creative coding.
### Waar loop je tegen aan?
Ik moet nog met Mariska het emailadres regelen van het form, ik was bij de sprint review vergeten te vragen of 250 submissions per maand genoeg is, anders moet ik nog een andere manier gaan instellen. Ik weet niet of ik haar zelf moet mailen hierover, teams, langs moet komen, of via jou Nazneen?

**Comment by Lmikkers** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?
Het gaar wel oke maar zit in een dipje en mijn motivatie is erg laag, moet weer even uit de sleur komen. Het is wel fijn dat ik nu weer op de opdrachten kan richten en ga dit ook doen, dus een inhaalactie.

### Wat wil je deze week bereiken (aan werken)?
Ik wil mijn deeltaak weer verder oppakken en wil issues inschieten voor de taken van bieb in bloei en een planning maken, zodat ik bieb in bloei niet vergeet te doen.

### Waar loop je tegen aan?
Dat ik achterloop, maar dit ga ik inhalen!
